geomesa-convert/geomesa-convert-avro/src/test/scala/org/locationtech/geomesa/convert/avro/Avro2SimpleFeatureConverterTest.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.convert.avro
+
+import com.typesafe.config.ConfigFactory
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.convert.SimpleFeatureConverters
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class Avro2SimpleFeatureConverterTest extends Specification with AvroUtils {
+
+  "Avro2SimpleFeature should" should {
+
+    val conf = ConfigFactory.parseString(
+      """
+        | converter = {
+        |   type   = "avro"
+        |   schema = "/schema.avsc"
+        |   sft    = "testsft"
+        |   id-field = "uuid()"
+        |   fields = [
+        |     { name = "tobj", transform = "avroPath($1, '/content$type=TObj')" },
```

Comment:
The AvroPath stuff?  It's intended to look like xpath but for traversing an Avro GenericRecord object.